README.md

```diff
@@ -104,9 +104,9 @@ CLI_E2E_ORG_ID = <organization_id>
 npm run test:e2e
 ```
 
-#### or using AVA:
+#### or using jest:
 ```sh
-$ ava test/integration/** --verbose --serial
+$ jest test/integration/** --verbose --serial
```

Comment:
shouldn't this be `npm run test:integration` or something similar?

Reply:
yea i could do `npm run test:e2e`, or i would consider making a command for just running them locally without --runInBand (see my comment on package.json)